src/main/docker/Dockerfile.jvm

```diff
@@ -21,34 +21,36 @@
 # docker run -i --rm -p 8080:8080 -p 5005:5005 -e JAVA_ENABLE_DEBUG="true" quarkus/release-manager-jvm
 #
 ###
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
```

Comment:
Should we mention the version instead of using latest ? Why: As the minimal image dont include JDK or maven tools, then we can go ahead and use latest to always get the updated installed packages (curl, bash, lib, ...) : https://catalog.redhat.com/software/containers/ubi8-minimal/5c64772edd19c77a158ea216?gti-tabs=unauthenticated&container-tabs=packages

Reply:
Ok, I don't have an opinion against it

Reply:
What do you then decide ? @aureamunoz

Reply:
`latest` version